ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -104,8 +95,11 @@ private static void createBuses(UcteNetworkExt ucteNetwork, UcteVoltageLevel uct
 
             Bus bus = voltageLevel.getBusBreakerView().newBus()
                     .setId(ucteNodeCode.toString())
+                    .setV(ucteNode.getVoltageReference())
```

Comment:
Some nodes don't have a generator. So if we don't store the voltageReference when importing we lose that information (or it is somewhere else ?).

Reply:
Bu if there is no generator, there is no voltageReference?

Reply:
Well, if I import a uct and then export it, some of the nodes that had a voltageReference before importing don't have them back when exporting because they don't have generator. Maybe it is an import problem then ?